### ISSUE TITLE
Add ability to send messages to player (playersocket)

### ISSUE THREAD

**Issue by spirodonfl** [ryanwinchester/spirodon_games#1]
Messages we want to be able to send to individual players

* General success failure messages for
* - spawn
* - attack
* - up
* - down
* - left
* - right
* Leaderboard information for specific user
* Current location (x/y) (as long as you're spawned in)
* Who's turn is it
* Current players
* Current HP (if spawned in)
* - just something to consider, we may want to send a "stats" object where we ship more than just HP, for example, things like, whether a buff is on, how much range your ship currently has, etc...

**Comment by ryanwinchester** [comment]
So do you want to
- send specific messages to specific players or
- broadcast diffs or events or something to all players to update the game state for all of them?

**Comment by spirodonfl** [comment]
Is it possible to do both?

**Comment by ryanwinchester** [comment]
Yes

**Comment by spirodonfl** [comment]
Are both easy or is one really difficult?

**Comment by ryanwinchester** [comment]
Yes

**Comment by ryanwinchester** [comment]
Okay i'll just make a pass-through for two message types

```json
{
  "send": {
    "player": "",
    "payload": {}
  }
}
```

```json
{
  "broadcast": {
    "payload": {}
  }
}
```

**Comment by ryanwinchester** [comment]
Closed in [2cf3349](https://github.com/ryanwinchester/twitch_gameserver/commit/2cf3349dbe4fbd05ad9edccc326dca5cfa8e127a)

**Comment by ryanwinchester** [comment]
## Note, in this commit:

## Changes

For player sockets, they used to send commands like this
```json
 {
   "command": "",
   "user": {},
   "ts": 1234567
}
 ```
but it becomes this
```json
{
  "cmd": "",
  "ts": ""
}
```

Also the connect path is now
```
/playersocket?channel=spirodonfl&id=123&login=foobar&name=FooBar
```

where `id`, `login`, and `name` are the user's twitch fields `id`, `user_login`, and `display_name`
